aws/ssm/plugin_windows.go

```diff
@@ -0,0 +1,5 @@
+package ssm
+
+func getSessionManagerPluginPath() (string, error) {
+	return "session-manager-plugin", nil
```

Comment:
If all of these are the same, what's the point of having different ones?

Reply:
Added some logic to increase chances of finding plugin binary.